advntr/vntr_finder.py

```diff
@@ -499,7 +502,16 @@ def get_dominant_copy_numbers_from_spanning_reads(self, spanning_reads):
             if logp < rev_logp:
                 vpath = rev_vpath
             observed_copy_numbers.append(get_number_of_repeats_in_vpath(vpath))
-
+            if log_pacbio_reads:
+                repeats = get_number_of_repeats_in_vpath(vpath)
+                logging.debug(str(Seq(haplotype)))
+                visited_states = [state.name for idx, state in vpath[1:-1]]
+                if self.read_flanks_repeats_with_confidence(vpath):
```

Comment:
This requires two arguments; read_sequence is missing.

Reply:
Added the read_sequence argument.